01_reverse.rb

```diff
@@ -15,5 +15,6 @@
 #
 
 def reverse (string)
-  # Your code here
+  string = "hello"
```

Comment:
It should work without this line.

If you keep this line, 
`reverse("travis")` returns `olleh`